# Header - frame_id & timestamp
Header header
# Vector of Lane Lines
LaneLine[] lane_lines